docs/ai/quickstarts/get-started-mcp.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Get started with .NET AI and MCP
+description: Learn about .NET AI and MCP key concepts and development resources
+ms.date: 04/29/2025
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: fboucher
+ms.author: frbouche
+# CustomerIntent: As a .NET developer new to OpenAI, I want deploy and use sample code to interact to learn from the sample code to summarize text.
+---
+
+# Get started with .NET AI and the Model Context Protocol
+
+The Model Context Protocol (MCP) is an open protocol designed to standardize integrations between AI apps and external tools and data sources. By using MCP, developers can enhance the capabilities of AI models, enabling them to produce more accurate, relevant, and context-aware responses.
+
+For example, using MCP, you can connect your LLM to resources such as:
+
+- Document databases or storage services
+- Web APIs that expose business data or logic
+- Tools that manage files or performing local tasks on a user's device
+
+Many Microsoft products already support MCP, including:
+
+- [Copilot Studio](/microsoft-copilot/blog/copilot-studio/introducing-model-context-protocol-mcp-in-copilot-studio-simplified-integration-with-ai-apps-and-agents)
+- [Visual Studio Code GitHub Copilot agent mode](https://code.visualstudio.com/blogs/2025/02/24/introducing-copilot-agent-mode)
+- [Semantic Kernel](https://devblogs.microsoft.com/semantic-kernel/integrating-model-context-protocol-tools-with-semantic-kernel-a-step-by-step-guide/).
+
+You can use the [MCP C# SDK](#develop-with-the-mcp-c-sdk) to quickly create your own MCP integrations and switch between different AI models without significant code changes.
+
+### MCP client-server architecture
+
+MCP uses a client-server architecture that enables an AI-powered app (the host) to connect to multiple MCP servers through MCP clients:
+
+- **MCP Hosts**: AI tools, code editors, or other software that enhance their AI models using contextual resources through MCP. For example, GitHub Copilot in Visual Studio Code can act as an MCP host and use MCP clients and servers to expand its capabilities.
+- **MCP Clients**: Clients used by the host application to connect to MCP servers to retrieve contextual data.
+- **MCP Servers**: Services that expose capabilities to clients through MCP. For example, an MCP server might provide an abstraction over a REST API or local data source to provide business data to the AI model.
+
+The following diagram illustrates this architecture:
+
+:::image type="content" source="../media/mcp/model-context-protocol-architecture-diagram.png" alt-text="A diagram showing the architecture pattern of MCP, including hosts, clients, and servers.":::
+
+MCP client and server can exchange a set of standard messages:
+
+|Message  |Description  |
+|---------|---------|
+|InitializeRequest     |  This request is sent by the client to the server when it first connects, asking it to begin initialization       |
```

Comment:
The message names look like they should be code-fenced.
And I think the descriptions should end in periods.